### DISCUSSION TITLE
Longer videos always end up with a moving texture

### DISCUSSION THREAD

**Discussion by Max-Basedaf** [deforum-art/sd-webui-text2video#164] [Q&A]
Hi all, I apologize if this issue has already been discussed, but I couldn't find any relevant information regarding it. I'm encountering a problem while generating videos, and I was hoping someone could assist me. Typically, when I generate videos with a frame rate of around 30fps, everything works smoothly. However, when I attempt to create longer videos, I often notice the presence of a moving "texture" pattern. To provide some context, I am using a 3090 graphics card, which should have sufficient VRAM to handle longer videos. Also [+] xformers version 0.0.20.dev526 installed. [+] torch version 2.0.0+cu118 installed.

Anyone knows how to solve this? Thank you in advance for your assistance!

https://github.com/deforum-art/sd-webui-text2video/assets/117739185/098a0cde-4ca8-4c03-8864-41fa0e24b113

**Comment by hithereai** [comment]
I don't think it's possible to "solve it" with the current model and architecture. 

Just make shorter vids and stitch them together in post. I believe this is what most (if not all) users are doing with ModelSc atm.

**Comment by B34STW4RS** [comment]
I had this happen to me on a few videos that shouldn't have been possible to generate. The solution in my case was a very high cfg, 15-20, and alot of steps, around 76 seemed to work good, secondly it would only work when eta was 1, finally I would increment the seed by 100 starting from 1000, some seeds gave moving texture some gave desired result. However, as hitherai said, its not really a solution, and is definitely not worth the trouble.

oh yeah and the parameters I used were 320x320 trying to do 60 frames at 12 fps. Though that had minimal impact on what I was trying to achieve.

**Comment by pinballelectronica** [comment]
I have the same issue-  I don't see a correlation to the video length though.  I can generate a video that will be largely clear and then the next one will have "grid lines" and wavy lines like a weave pattern.  Longer videos don't solve it for me. 


![000010](https://github.com/kabachuha/sd-webui-text2video/assets/122552749/e23abe02-02f4-45d3-9893-c7e01e0be134)
![ex1](https://github.com/kabachuha/sd-webui-text2video/assets/122552749/0685b155-4aa8-45dc-b3fe-8bdce5c2b235)
![000000](https://github.com/kabachuha/sd-webui-text2video/assets/122552749/ba4352f9-f315-4e29-a183-bc81edf09342)

**Comment by Max-Basedaf** [comment]
So you will get the moving pattern thing when you are reaching the capabilities of the trained model. So for instance generating a long video won't always work cause the models isnt trained on longer dataset videos. And sometimes the model doesn't understand the prompt cause it hasnt been trained enough to understand everything. When you wil get a moving pattern thingy you could try to describe your prompt in other words + i would advise keeping the maximum frames per clip low to get the best output for now.

**Comment by B34STW4RS** [comment]
To add to the confusion even more, some seeds have the moving texture, some seeds are fine, some seeds are extremely zoomed in, some are fine.

**Comment by andzejsp** [comment]
when we getting better models?
